README.md

```diff
@@ -1 +1,23 @@
 # hw01
+
+## **About me**
+
+* Name: Katerina Townsend
+
+* Age: 22
+
+* From: Atlanta, Georgia
+
+* Program: MAPSS, Psychology
+
+## **Fun facts**
+
+* I was once a state ranked archer
+
+* I worked on lots of movies and tv shows in Atlanta
+
+* I was an RA throughout college
+
+Connect with me on linkedin!
+
+[LinkedIn](www.linkedin.com/in/katerina-townsend-5a3254150)
```

Comment:
As someone else commented, this link sent me to a 404 error page as well, so this would be something to investigate.